packages/11ty/_includes/components/object-filters.webc

```diff
@@ -0,0 +1,37 @@
+<script webc:type="js">
+const { filters } = this.webc.attributes;
+const filterNames = filters
+  .split(',')
+  .map((filterName) => filterName.trim());
+
+const getFilterElements = () => filterNames
+  .map((filterName) => {
+    const filterOptions = objects.reduce((uniqueOptions, object) => {
+      const existingLabels = uniqueOptions.map(({ label }) => label);
+      const label = object[filterName];
+      const value = object[filterName];
+      if (!existingLabels.includes(label) && value) {
```

Comment:
the `&& value` added here on L13 adds basic error handling for object filters -- any filter name specified on the objects catalog parent component (`<objects-catalog filters="artist, medium"></objects-catalog>`) will render, but if no objects have that corresponding property, it will render a `<select>` with the specified label, but only a single "All" (`value: ''`) option present. This could be refined further to _not_ even render a filter field that isn't relevant to any of the current objects, but it will not break anything in the current state